### ISSUE TITLE
Amplitude of waves regularly go off screen after v1.1 of Waves Intro

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/wave-interference#527]
**Test device**
Dell
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
Seen during https://github.com/phetsims/qa/issues/770 and while taking screenshots. Happens in intro as well. Appeared in v1.1 of Waves Intro. May not happen in Wave Interference (at least not as obviously). Wave Interference is more likely to see this if there are two sources added together. May be connected to https://github.com/phetsims/wave-interference/issues/485.

When the graph tool is placed near the source of a wave (or even a fair distance away) the wave goes off the tools screen with the default settings.

**Visuals**
Before change:
<img width="959" alt="smallamplitude" src="https://user-images.githubusercontent.com/41024075/151460020-3472c61a-16d0-4630-80d7-46ee14cb9df7.png">
After change:
<img width="960" alt="largeamplitude" src="https://user-images.githubusercontent.com/41024075/151460472-dca9ebce-6f3c-4aea-8e03-75281beb5f90.png">

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Waves Intro‬
URL: https://phet-dev.colorado.edu/html/waves-intro/1.1.5/phet/waves-intro_all_phet.html
Version: 1.1.5 2021-12-15 23:09:26 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.36
Language: en-US
Window: 1280x649
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by samreid** [comment]
@KatieWoe and I discussed the behavior above and it seems like the next step is for @arouinfar to recommend how to proceed.

**Comment by arouinfar** [comment]
Looks like we should consider rescaling the chart tool. I don't think it's urgent, so it can wait until the next time this sim is republished. Unassigning myself for now.

**Comment by samreid** [comment]
For me, the behavior is reasonable and clearly indicates "there is more wave off the chart here", I'm not convinced that adding zoom buttons or rescaling will be the best solution. The user does not need to measure so close to the source and if they do it's unlikely to cause confusion.  This is a qualitative tool. So I recommend closing as won't fix. @arouinfar what's your take?

**Comment by arouinfar** [comment]
I agree @samreid. Also, the new scaling makes it easier to see the signal for low amplitudes, which seems like a benefit. Closing.
